Spanner/src/Connection/Grpc.php

```diff
@@ -455,6 +457,39 @@ public function createSession(array $args)
         ]);
     }
 
+    /**
+     * @return PromiseInterface
```

Comment:
Could we please use the same docblock as found [here](https://github.com/googleapis/google-cloud-php/blob/c3aa1ba6d60d8ae78e1b225bd087b4a8b781eb81/Spanner/src/Connection/Grpc.php#L521-L527).